target/linux/ath79/patches-6.6/317-MIPS-pci-ar724x-unset-power-down-of-pll-on-AR934x.patch

```diff
@@ -0,0 +1,23 @@
+--- a/arch/mips/include/asm/mach-ath79/ar71xx_regs.h
++++ b/arch/mips/include/asm/mach-ath79/ar71xx_regs.h
+@@ -347,6 +347,8 @@
+ #define AR934X_PLL_CPU_DDR_CLK_CTRL_DDRCLK_FROM_DDRPLL	BIT(21)
+ #define AR934X_PLL_CPU_DDR_CLK_CTRL_AHBCLK_FROM_DDRPLL	BIT(24)
+ 
++#define AR934X_PLL_PCIE_CONFIG_PLL_PWD		BIT(30)
```

Comment:
Please move this definition after `AR724X_PLL_REG_PCIE_CONFIG_PPL_RESET`.

Reply:
IMO, this definition shoud be placed into the AR934X PLL group.